### DISCUSSION TITLE
CardDAV account does not show up to be selected as "Default Account" in iOS

### DISCUSSION THREAD

**Discussion by paranerd** [Kozea/Radicale#1637] [Q&A client]
Hi all,

I successfully set up the radicale server and connected to it from my iPhone (14.7.1).
Now I can't select it as the "Default Account" (only iCloud and GMail show up).

This might not be an issue with radicale at all, I'm just glad for any suggestions on how to approach this.

Thank you!

**Comment by webscopeny** [comment]
Hi Paranerd,

Did you ever get this working? We're having a similar issue

David

**Comment by paranerd** [comment]
I did.

I don't quite remember what the issue was, but I think it had something to do with the URL.

For me it has the following format:
https://[url-to-my-server]/[username]/[some-long-hex-id]/

Maybe it was the trailing slash...
If you get to figure it out, please post here (as I should have) for posterity^^
